### PR TITLE
Implements orders list view

### PR DESCRIPTION
**Pull request by maximest-pierre**
_Fixes #197_ 

### Changes proposed in this pull request:

*  add test 
* add function to calculate the total of the order inside the model as a property
* add a view list for all the order of a given client
* add a list for all the order inside a system
* add link to the user by clicking his name inside the list view

### Status

- [x] READY
- [ ] HOLD
- [ ] WIP (Work-In-Progress)

### Deployment notes and migration

- [x] Migration needed

@savoirfairelinux/mll

### CODE REVIEW

**Comment by maximest-pierre** [comment]
@manumilou Can you check this PR and tell me if there is any modification I need to make on it.

File: django/santropolFeast/order/models.py [hunk 1]
```diff
@@ -50,9 +53,65 @@ class Meta:
 
     client = models.ForeignKey(
         'member.Client',
-        verbose_name=_('client')
+        verbose_name=_('client'),
+        related_name='client_order',
     )
 
+    @property
+    def total(self):
```
Comment (manumilou) [inline django/santropolFeast/order/models.py]: Change total for: **price**

File: django/santropolFeast/order/templates/list_all_order.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{% extends "base_order.html" %}
```
Comment (manumilou) [inline django/santropolFeast/order/templates/list_all_order.html]: Change the template name for: list.html

File: django/santropolFeast/order/templates/list_order.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
+{% extends "base_order.html" %}
```
Comment (manumilou) [inline django/santropolFeast/order/templates/list_order.html]: What's the difference with list_all_order.html ?

Comment (maximest-pierre) [inline django/santropolFeast/order/templates/list_order.html]: One them is the old template I am going to clean it and commit the changes

Comment (manumilou) [inline django/santropolFeast/order/templates/list_order.html]: Please rename the template for a simpler: **list.html**

**Comment by maximest-pierre** [comment]
@manumilou Everything is good now

**Pull request merged**